src/core/Global.ttslua

```diff
@@ -614,21 +622,17 @@ end
 -- Content Importing and XML functions
 ---------------------------------------------------------
 
-local source_repo = 'https://raw.githubusercontent.com/chr1z93/loadable-objects/main'
-local library = nil
-local request_obj
-
 function onClick_refreshList()
-  local request = WebRequest.get(source_repo .. '/library.json', completed_list_update)
-  request_obj = request
+  local request = WebRequest.get(SOURCE_REPO .. '/library.json', completed_list_update)
+  requestObj = request
```

Comment:
Please try and keep no-op refactorings in separate PRs

Reply:
Will do 🫡